build_scripts/docker_image_creator.py

```diff
@@ -0,0 +1,59 @@
+"""
+Script to run docker image creation
+Run the command:
+    python docker_image_creator.py optimized_binary_link gpu|cpu tensorflow_version python_version
+"""
+import argparse
+import os
+import sys
+
+## GLOBALS
+# Allows user to run script anywhere
+BASE_PATH = ""
+
+def create_docker_image(optbin_link, processor, framework_version, python_version):
+    """
+    Function builds a docker image with the TF optimized binary
+
+    :param optbin_link: link to where the optimized binary is
+    :param processor: gpu or cpu
+    :param framework_version: tensorflow version i.e 1.6.0
+    :param python_version: version of python to build container with i.e. 3.6.5 or 2.7.4
+    """
+    # 1.) Initialize some commonly used variables
+    pyV = "py{}".format(python_version.split('.')[0]) # i.e. py2
+    framework = "tensorflow"
+    # 2.) Get optimized binary - and put in final docker image repo
```

Comment:
just a suggestion - it might help to add newlines between the different steps for readability